src/assets/img/devices/jellybook.svg

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You can export an SVG in Inkscape as optimized so it doesn't contain information for editing, additionally a tool like `svgo` can optimize the SVG even more to decrease the file size.

Reply:
Hi, I just saved the svg as a optimized svg then optimized it with `svgo`